tests/src/test/scala/whisk/core/loadBalancer/test/InvokerSupervisionTests.scala

```diff
@@ -206,7 +206,8 @@ class InvokerSupervisionTests
 
     sendActivationToInvoker
       .when(activationMessage, invokerInstance)
-      .returns(Future.successful(new RecordMetadata(new TopicPartition(invokerName, 0), 0L, 0L, 0L, 0L, 0, 0)))
+      .returns(Future.successful(
+        new RecordMetadata(new TopicPartition(invokerName, 0), 0L, 0L, 0L, java.lang.Long.valueOf(0L), 0, 0)))
```

Comment:
Fairly interesting interface 🤔 

There's a scala version for `java.long.Long.valueOf` --> `Long.box(0L)`

Reply:
yeah, someone is playing a game with overloading and deprecation.  Must be a story behind it in kafka land...